src/controller/python/templates/python-cluster-Objects-py.zapt

```diff
@@ -56,6 +56,17 @@ class {{asUpperCamelCase name}}(Cluster):
 
 {{/zcl_enums}}
 
+{{#zcl_bitmaps}}
+{{#first}}
+    class Enums:
+{{/first}}
+        class {{asType label}}(IntEnum):
```

Comment:
Curious, why are you not using `IntFlag`? Would you not want a value of this type to be a combination of these flags?

Reply:
thanks, updated.